app-creativity/rnote/spec

```diff
@@ -0,0 +1,5 @@
+VER=0.11.0
+SRCS="tbl::https://github.com/flxzt/rnote/archive/v$VER.tar.gz"
```

Comment:
Please use a Git tag per the [Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/), we do not allow using platform-generated tarballs as the checksums may not be stable over time.